habitat-hitl/habitat_hitl/_internal/hitl_driver.py

```diff
@@ -538,10 +541,11 @@ def depth_to_rgb(obs):
                     if "rigUpdates" in keyframe_obj:
                         del keyframe_obj["rigUpdates"]
                 # Insert server->client message into the keyframe
-                message = self._client_message_manager.get_message_dict()
+                # TODO: Only one user is currently supported.
+                message = self._client_message_manager.get_messages()[0]
```

Comment:
Should the `0` be a variable ("current_user" or something) instead of hard-coded to prepare for expansion?

Reply:
The concept of `current_user` will be controlled by the application itself (user code). This specific snippet will always provide all user messages to the networking thread.